sharness/__snapshots__/test-instance/config/dependencies.json

```diff
@@ -4,7 +4,11 @@
         "autoInjectStartingPeriodWeight": 0.05,
         "id": "2zWwkGHnBGXmEcYneeWAzw",
         "name": "SourceCred",
-        "periods": [{"startTime": "2019-01-01", "weight": 0.05}]
+        "periods": [
+            {
+                "startTime": "2019-01-01",
+                "weight": 0.05
+            }
+        ]
     }
-]
-
+]
```

Comment:
Why these changes, especially the missing newline at end of file? They
look unrelated to this change, and `yarn test` passes for me without
them. Please fix?

Reply:
Its required for yarn test-full. It was slightly inaccurate for me to say this PR doesn't change behavior. Prior to this change, writes to the configuration would only happen if the configuration had been changed by the code, but now the configuration always is written. I chose this because it made the code simpler and we can put that optimization back in later if we want. But this means that the expected snapshot needs to match the computer-generated json. This file was edited for this PR by running the update-snapshots script.

Reply:
Ack; thanks.